config/paths.js

```diff
@@ -5,8 +5,11 @@ const fs = require('fs');
 const url = require('url');
 
 const ASSETS = 'assets';
+const ASSETS_FOLDER = 'assets';
+const DIST_FOLDER = 'dist';
 const SOURCE_FOLDER = 'src';
-const appSrc = `./${ASSETS}/${SOURCE_FOLDER}`;
+const ASSETS_DIST_FOLDER = `${ASSETS_FOLDER}/${DIST_FOLDER}`;
+const ASSETS_SOURCE_FOLDER = `${ASSETS_FOLDER}/${SOURCE_FOLDER}`;
```

Comment:
As per this work done before: https://github.com/eventespresso/event-espresso-core/pull/2609/files#diff-e389ed4107fbd6139ae8b315cb04e04d